src/pretix/control/forms/organizer.py

```diff
@@ -211,6 +211,16 @@ class OrganizerSettingsForm(SettingsForm):
         widget=I18nTextarea,
         help_text=_('Not displayed anywhere by default, but if you want to, you can use this e.g. in ticket templates.')
     )
+
+    event_team_provisioning = forms.BooleanField(
+        label=_('Allow team provisioning for newly created events.'),
+        help_text=_('Users that do not have access to all events under this organizer, must select one of their teams '
+                    'to have access to the created event. This setting allows users to create an event-specified team'
+                    'on-the-fly.'),
+        required=False,
+        initial=True,
```

Comment:
initial=True won't do it for existing organizers. if it stays, we need to define the default value in pretix/base/settings.py instead.

Reply:
I've moved the initial defaults to pretix/base/settings.py